TTIEX355P/iocBoot/iocTTIEX355P-IOC-01/st.cmd

```diff
@@ -55,10 +55,12 @@ $(IFNOTFIELD_CONV_DEFINED) epicsEnvSet("AMPSTOGAUSS", "1")
 $(IFNOTFIELD_CONV_DEFINED) epicsEnvSet("FIELD_CONV_DEFINED", "0")
 $(IFFIELD_CONV_DEFINED) epicsEnvSet("FIELD_CONV_DEFINED", "1")
 
+stringiftest("DOAUTOSAVESETPOINTS", "$(AUTOSAVESETPOINTS=NO)", 5, "YES")
+
```

Comment:
Do you need "DO" ?  I feel  `IFAUTOSAVESETPOINTS`   sounds nicer than   `IFDOAUTOSAVESETPOINTS`